tables/subject_history/ikemura21-pth.csv

```diff
@@ -0,0 +1,7 @@
+,ref_name,subject_name,event,event_date_cmp,event_date,location,iso_name,vaccine_name,severity
+0,Ikemura21,B.1.1.7_infection,isolation,~,2021-01-31,Japan,NULL,NULL,NULL
+1,Ikemura21,B.1.1.7_infection,isolation,~,2021-01-31,Japan,NULL,NULL,NULL
```

Comment:
I think the event_date for all 3 isolations are not right because the paper said they get plasma 3m after infection or 2nd dose. For example `B.1.1.7_infection` the infection is `2021-01-31`, then the estimated isolation data could be `2021-04-31`.